compiler/damlc/tests/daml-test-files/NewTemplateDesugaring.daml

```diff
@@ -0,0 +1,13 @@
+daml 1.2
+
+module NewTemplateDesugaring where
```

Comment:
I guess we can kill this test now that you’re switching over everything?

Reply:
Killed!